manifests/istio-control/istio-discovery/templates/telemetryv2_1.5.yaml

```diff
@@ -219,6 +219,7 @@ spec:
                   {
                     "debug": "false",
                     "stat_prefix": "istio",
+                    "disable_host_header_fallback": true,
```

Comment:
We'll need a release note since this is backwards incompatible.

Reply:
Yep

Reply:
Just to confirm, by backwards incompatibility you mean there will no longer be host header fallback for request that hits NR/passthrough at gateway, right? Or is there some other incompatibility?

Reply:
Yeah, there'll be a change in telemetry. I think that means some metrics will lose destination service in some cases.